.codeclimate.yml

```diff
@@ -1,3 +1,6 @@
+engines:
+  rubocop:
+    enabled: true
```

Comment:
We already run RuboCop as a part of our default Rake task in Travis, I don't think we need it here